### ISSUE TITLE
ResNet doesn't have softmax layer in last layer.

### ISSUE THREAD

**Issue by PeterKim1** [kuangliu/pytorch-cifar#132]
Hi. Thank you for your great codes.

I have a question for your codes.

In ResNet Paper, "The network ends with a global average pooling, a 10-way fully-connected layer, and softmax"(Page 7, 4.2 CIFAR-10 and Analysis)

Why your code don't have softmax layer in last layer?

**Comment by PeterKim1** [comment]
nn.CrossEntropyLoss has nn.LogSoftmax, so this model doesn't need to have softmax layer in last layer.
